superset/commands/dashboard/importers/v1/utils.py

```diff
@@ -182,8 +182,6 @@ def import_dashboard(
                 logger.info("Unable to encode `%s` field: %s", key, value)
 
     dashboard = Dashboard.import_from_dict(session, config, recursive=False)
-    if dashboard.id is None:
-        session.flush()
```

Comment:
Why is this being removed? We currently need it for `ImportAssetsCommand` and `ImportExamplesCommand`.